transport-classes-io_uring/src/main/java/io/netty/channel/uring/AbstractIoUringChannel.java

```diff
@@ -67,6 +68,7 @@
 
 abstract class AbstractIoUringChannel extends AbstractChannel implements UnixChannel {
     private static final InternalLogger logger = InternalLoggerFactory.getInstance(AbstractIoUringChannel.class);
+    private static final boolean USE_POLL_IN_FIRST = SystemPropertyUtil.getBoolean("io.netty.iouring.usePollInFirst", false);
```

Comment:
I was even thinking about an `IoUringChannelOption` that people can use before registration is done. @chrisvest

Reply:
I think so too. But I've found that actually users can't specify the option.
Because it's specified when we construct the IoUringSocketChannel.

``` java
    IoUringSocketChannel(Channel parent, LinuxSocket fd, SocketAddress remote) {
        super(parent, fd, remote);
        this.config = new IOUringSocketChannelConfig(this);
    }
```

Perhaps I could provide a pull request with more significant changes to allow users to specify the corresponding IOUringSocketChannelConfig.

Reply:
Let me take a stab. I think it is possible

Reply:
> Let me take a stab. I think it is possible

i have implement it by using channel option